code/public/index.html

```diff
@@ -1,7 +1,36 @@
 <!DOCTYPE html>
 <html lang="en">
-
   <head>
+    <!-- HTML Meta Tags -->
+    <title>Technigo React App</title>
+    <meta
+      name="description"
+      content="Todo app to help you keep your todo's on one place."
+    />
+
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="https://a1eksa-todo.netlify.app/" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Todo app" />
+    <meta
+      property="og:description"
+      content="Todo app to help you keep your todo's on one place."
+    />
+    <meta property="og:image" content="https://imgur.com/a/J0Q7Xb5" />
```

Comment:
The og tag for the image is not working. Think you have to copy and paste the image address (https://i.imgur.com/73VoDjN.png) instead of the link.